packages/apps/frontera/src/routes/organizations/src/components/SidePanel/SidePanel.tsx

```diff
@@ -0,0 +1,30 @@
+import { useSearchParams } from 'react-router-dom';
+
+import { useStore } from '@shared/hooks/useStore';
+
+import { Icp } from '../ICP';
+
+export const SidePanel = () => {
+  const store = useStore();
+
+  const [searchParams] = useSearchParams();
+
+  const preset = searchParams.get('preset');
+
+  const tableViewName = store.tableViewDefs.getById(preset || '')?.value.name;
+
+  return (
+    <div className='flex'>
+      {/* {tableViewName === 'Contacts' && (
```

Comment:
if is needed?

Reply:
yes since i started also the persona builder on this branch.